src/app/globals.css

```diff
@@ -62,5 +62,5 @@
 }
 
 .link {
-  @apply text-primary underline;
```

Comment:
❓ Wondering if we should change the value in tailwind.config.css instead - anyone know if there would be any downsides to doing that?

Reply:
I'm not familiar with Tailwind either, but happy to change Tailwind's config file if that makes more sense to everyone.

Reply:
I wasn't super familiar with Tailwind either before looking at this project, but they have some pretty good [docs for how to configure colors](https://tailwindcss.com/docs/customizing-colors).

Reply:
this is the right way to do in tailwind (to the best of my knowledge!)